python/gen3authz/client/arborist/client.py

```diff
@@ -547,6 +547,24 @@ def update_policy(self, policy_id, policy_json):
         self.logger.info("put policy {}".format(policy_id))
         return response
 
+    def upsert_policy(self, policy_id, policy_json):
```

Comment:
can we merge this into `update_policy` with `create=True` (default?) or something like that?